### PR TITLE
Adds moment date fields to PartnerShow

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

**Comment by dzucconi** [comment]
```
{
  partner_show(id: "new-museum-1-2015-triennial-surround-audience") {
    id
    start_at(format: "dddd, MMMM Do YYYY, h:mm:ss a")
    end_at(format: "YYYY")
  }
}

// =>
{
  "data": {
    "partner_show": {
      "id": "new-museum-1-2015-triennial-surround-audience",
      "start_at": "Wednesday, February 25th 2015, 7:00:00 am",
      "end_at": "2015"
    }
  }
}
```

**Comment by dzucconi** [comment]
Oh, UTC... good catch Semaphore.

File: schema/fields/date.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { GraphQLString } from 'graphql';
+
+export default {
+  type: GraphQLString,
+  args: {
+    format: {
+      type: GraphQLString
+    }
+  },
+  resolve: (show, { format }, { fieldName }) => {
```
Comment (mzikherman) [inline schema/fields/date.js]: maybe instead of show just event ? since it's gonna be used in fairs, etc

**Comment by mzikherman** [comment]
Looks sick! Great spec too.

Made one minor comment.

File: schema/fields/date.js (hunk 1 above)
Comment (dzucconi) [inline schema/fields/date.js]: Ah, woops thank you

**Pull request merged**